### ISSUE TITLE
Problem with quoted response code

### ISSUE THREAD

**Issue by lscalese** [lscalese/OpenAPI-Client-Gen#34]
When we upload YAML definition like that : 

```
paths:
  /test:
    get:
      responses:
        '200':
          description: OK
```

The conversion to JSON result : 

```
  "paths":{
    "/test":{
      "get":{
        "responses":{
          "'200'":{
            "description":"OK"
          }
        }
      }
    }
```
In JSON, the result should be `"200"`, not `"'200'"`.  
May be, it's a case to handle in [yaml-utils](https://openexchange.intersystems.com/package/yaml-utils) package.
To analysis...

**Comment by bdeboe** [comment]
oops, wrote in the wrong thread -- I'll merge the PR shortly 👍

**Comment by lscalese** [comment]
Solved with yaml-utils 0.1.3
